dev-lua/luv/luv-1.30.1.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils unpacker
+
+# e.g. PV = a.b.c.d
+# a.b.c
+MY_BASE="${PV%.*}"
+# d
+MY_PATCH="${PV##*.}"
+# a.b.c-d
+MY_PV="${MY_BASE}-${MY_PATCH}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="Bare libuv bindings for lua"
+HOMEPAGE="https://github.com/luvit/luv"
+# XXX: Remember to check this hash between bumps!
+# https://github.com/luvit/luv/tree/master/deps
+LUA_COMPAT_HASH="daebe77a2f498817713df37f0bb316db1d82222f"
+SRC_URI="
+	https://github.com/luvit/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/keplerproject/lua-compat-5.3/archive/${LUA_COMPAT_HASH}.tar.gz -> ${PN}-lua-compat-${PV}.tar.gz
+"
+
+LICENSE="Apache-2.0"
```

Comment:
Btw, check the license for lua-compat package, this needs updating :)
can't believe how badly I ignored the lua-compat thing altogether last time...

Reply:
Oh, it's MIT; do I just space-separate it `LICENSE="Apache-2.0 MIT"`?